doc/attr.xml

```diff
@@ -13,14 +13,14 @@
   <Attr Name="ChromaticNumber" Arg="digraph"/>
   <Returns> A non-negative integer.</Returns>
   <Description>
-    A digraph coloring is a labeling of the vertices (using precisely <A>n</A>
```

Comment:
Yes this must have been copied from the documentation for `DigraphColouring` when the documentation for `ChromaticNumber` was add. I'll reword it.